### DISCUSSION TITLE
Color names with id, RGB, HSL, hex code values & some color mixing utilities

### DISCUSSION THREAD

**Discussion by aMiss-aWry** [evennia/evennia#3145] [Community Contribs & Snippets]
https://github.com/aMiss-aWry/muddev-snippets

You'll also find a (janky) autocomplete plugin. 
The color mixing is a bit of a work in progress. 
Color names are sourced from the discord. Been revised over time. Happy to take suggestions.

**Comment by Griatch** [comment]
A nice resource! The only feedback is that the README doesn't actually explain how and in what way they could be used with Evennia.
